src/lib/i18n.json

```diff
@@ -100,6 +100,13 @@
     "deployment-table-add-src-to-acr-pipeline": "Could not add source to ACR pipeline information to storage table.",
     "deployment-table-add-acr-to-hld-pipeline": "Could not add ACR to HLD pipeline information to storage table.",
 
+    "ring-err-create": "Error occurred while creating ring: {0}",
+    "ring-err-exists": "Ring: {0} already exists in project {1}.",
```

Comment:
message should be like this.
`Could not create ring {0} in project {1} because it already existed. Provide a different name.`
The first sentence (in past tense) mentioning what had happened. The second sentence offer an suggestion on how to fix it. - https://github.com/CatalystCode/spk/blob/master/guides/command-implementation.md#error-messages